semantics.json

```diff
@@ -112,7 +112,39 @@
     "label": "Localization",
     "name": "l10n",
     "type": "group",
-    "fields": []
+    "common": true,
```

Comment:
Was `common` required for it to work?

Reply:
If we don't use `common` the translations will not be placed under the collapse "Text overrides and translations" but a collapse looking like this:
<img width="619" alt="Skjermbilde 2023-02-21 kl  10 48 14" src="https://user-images.githubusercontent.com/35261194/220309907-6670e4a6-a83f-4137-a130-456cc56744b0.png">

Reply:
But it does seems to work without `common`

Reply:
Ahhh common it is then!